lib/chai/core/assertions.js

```diff
@@ -2013,6 +2048,11 @@ module.exports = function (chai, _) {
    *     expect([1, 2, 3]).to.have.lengthOf(3);
    *     expect('foo').to.have.lengthOf(3);
    *
+   * When the target is a map or set, asserts that the target's `size` property.
```

Comment:
This reads awkwardly to me. Instead of having this separate line, I suggest updating lines 2045-2046 to read: "Asserts that the target's `length` or `size` is equal to the given number `n`."

And then the new examples could go right beneath the "foo" example.

Reply:
Thanks!